contracts/DNSRegistrar.sol

```diff
@@ -0,0 +1,133 @@
+pragma solidity ^0.4.23;
+
+import "./ens.sol";
```

Comment:
let's use the npm package for this please

Reply:
Removed